### ISSUE TITLE
Exclusion Zone structures are smaller than expected

### ISSUE THREAD

**Issue by Tanjiraaaaa** [kd8lvt/exclusionzone#4]
![2024-07-27_21 17 00](https://github.com/user-attachments/assets/636eaa0e-dc29-4818-a12b-5e2a14b753ea)
It's like a quarter of a chunk or smt
My version is 1.20.6 Fabric
Here's my [modlist](https://mclo.gs/Lf0huXh)

**Comment by kd8lvt** [comment]
That's just what the structure looks like :) You might want to take a gander around and inside of it, though - not everything is simple decor~

**Comment by Tanjiraaaaa** [comment]
> That's just what the structure looks like :) You might want to take a gander around and inside of it, though - not everything is simple decor~

Everything around it is a different biome though 🤔

**Comment by Tanjiraaaaa** [comment]
The biome is supposed to be that small?

**Comment by kd8lvt** [comment]
Yep, there's some loose lore (more like rough design threads) that I try to adhere to - though I'm not currently looking to release it, I'd rather people come up with their own ^-^
Tl;dr - the monoliths aren't from your world, they're from a parallel dimension (as in like, 2D/3D/4D/etc - not the Nether.) The exclusion zone biome is a side-effect of the incursion.

**Comment by kd8lvt** [comment]
There are a couple other structures in the mod, which also have that idea behind them. When exploring, it's worth taking a Brush around with you, as Exclusion Zone makes heavy use of the vanilla Archaeology system :) (Just make sure you have a decent amount of inventory space, there's a lot of unique items)
I'll be writing a wiki at Some Point:tm: here on GitHub that'll go in-depth as to how you get certain items and what all they do, but a good amount of the fun is discovering things on your own so I'm not sure how I'd go about it.

**Comment by kd8lvt** [comment]
I'll close this as completed, though I'm pinning it as there's some good info here for folks.

**Comment by Tanjiraaaaa** [comment]
I see, thank you !

**Comment by kd8lvt** [comment]
You're welcome! I'm glad you're enjoying the mod :)
